source/features/highest-rated-comment.css

```diff
@@ -14,12 +14,16 @@ a.rgh-highest-rated-comment .avatar {
 	left: -55px;
 }
 
-.rgh-highest-rated-comment {
-	--color-auto-gray-1: var(--color-auto-green-1);
-	--color-auto-blue-4: var(--color-auto-green-4);
-	--color-auto-blue-2: var(--color-auto-green-2);
+.rgh-highest-rated-comment:before,
+.rgh-highest-rated-comment:after {
+	--color-bg-info-inverse: var(--color-text-success, #3fb950);
+	--color-bg-tertiary: var(--color-text-success, #3fb950);
+	left: initial;
```

Comment:
This should not be necessary I think

Reply:
It unfortunately is, the green caret appears black otherwise

Reply:
I realise you might be speaking of `left: initial` which fixes the caret positioning

Reply:
```suggestion
```

Reply:
This is a color change. If GitHub can style the arrow correctly it means its position is correct.


<img width="97" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/125120001-0be7b300-e11c-11eb-8147-1c708713bce0.png">

Also, it's not being applied at all:

<img width="579" alt="Screen Shot 7" src="https://user-images.githubusercontent.com/1402241/125119559-7a784100-e11b-11eb-8941-581d6c0f5c36.png">